packages/code-du-travail-data/dump.js

```diff
@@ -39,12 +40,15 @@ async function fetchVector(data) {
 const dump = async () => {
   let documents = [];
   if (NLP_URL) {
-    console.error(`use ${NLP_URL} to retrieve tf vectors`);
+    console.error(`Using NLP service to retrieve tf vectors on ${NLP_URL}`);
   } else {
-    console.error(`no nlp`);
+    console.error(`NLP_URL not defined, semantic search will be disabled.`);
   }
-  for await (const docs of cdtnDocumentsGen()) {
-    console.error(`› ${docs[0].source}... ${docs.length} items`);
+  for await (const docsRaw of cdtnDocumentsGen()) {
+    console.error(`› ${docsRaw[0].source}... ${docsRaw.length} items`);
+    // add covisits
+    const docs = await Promise.all(docsRaw.map(fetchCovisits));
```

Comment:
you should add a concurrency param here (using p-limit for example) to avoid making 300+ request in // depending  on the size of the documents